src/mokka/src/integration-test/java/pl/hycom/mokka/emulator/mock/MockConfigurationControllerIT.java

```diff
@@ -0,0 +1,180 @@
+package pl.hycom.mokka.emulator.mock;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.web.context.WebApplicationContext;
+import pl.hycom.mokka.Application;
+
+import java.util.UUID;
+import java.util.stream.StreamSupport;
+
+import static io.restassured.RestAssured.expect;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+/**
+ * @author adam.misterski@hycom.pl
+ */
+@DirtiesContext
+@ActiveProfiles("test")
+@AutoConfigureMockMvc
+@AutoConfigureTestDatabase
+@SpringBootTest(classes = Application.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class MockConfigurationControllerIT {
+
+    @Autowired
+    protected WebApplicationContext wac;
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    private MockConfigurationRepository mockConfigurationRepository;
+
+    @Autowired
+    private WireMockServer wm;
+
+    private static final String name = "test";
+    private static final String httpMethod = "GET";
+    private static final int status = 200;
+    private static final String path = "path";
+
+    private static final String CONTENT_TYPE_KEY = "Content-Type";
+
+    @BeforeEach
+    public void setUp() throws Exception{
+        wm.start();
```

Comment:
You do not need to start WireMock server manually. Application should done it itself. What you need to do is to change wiremock server configuration as by default it is disabled. The simplest solution would be:
`@SpringBootTest(classes = Application.class, properties = {"wiremock.enabled=true"})